cmd.go

```diff
@@ -139,14 +139,14 @@ func Run(request transform.PluginRequest) (transform.PluginResponse, error) {
 	case "RoleBinding":
```

Comment:
I think we need to drop everything that has `authorization.openshift.io`, right?

Reply:
Yes. I don't know why I got so hyper focused on RoleBindings, but ot does look like there are a handful of others as you'd expect (Roles, ClusterRoles - admittedly cluster scoped, etc.). I'll fix this to white them all out. Thanks for catching this.